packages/common/src/initiatives/_internal/InitiativeSchema.ts

```diff
@@ -12,5 +17,24 @@ export const InitiativeSchema: IConfigurationSchema = {
   type: "object",
   properties: {
     name: ENTITY_NAME_SCHEMA,
+    summary: {
+      type: "string",
+    },
+    description: {
+      type: "string",
+    },
+    access: ENTITY_ACCESS_SCHEMA,
+    groups: {
```

Comment:
I don't know how helpful abstracting the shared schemas out actually is, and I didn't do it for the super basic ones like summary/description, etc., but we could abstract this one into the shared directory since we're using it for projects, initiatives, and sites.

I think we'll also eventually want to do the same for the location schema and actually provide a more robust schema definition to provide appropriate validation. Something like: 

```ts
{
  type: "object",
  properties: {
    type: {
      type: "string",
      enum: ["none", "custom", "org", "item"]
    },
    extent: {
      type: "array",
      items: {
        type: "array",
        items: {
          type: "number"
        }
      }
    },
   ...
  }
}
```